versions.config

```diff
@@ -8,7 +8,7 @@ RAW_IMAGE_VERSION="v0.2.1"
 RAW_IMAGE_CHECKSUM="0fe529a329e0fe51fa0bff0aadf63f17b30f3db6b9fa882431fe51c1ae5f9302"
 
 # specific versions of kernel/firmware and docker tools
-export KERNEL_BUILD="1.20190215-1"
+export KERNEL_BUILD="1.20190401-1"
```

Comment:
Nice catch, thanks!

Reply:
> Nice catch, thanks!

Yeah, for some reason the 1.20190215 stopped working about a week ago...  I think that artifact disappeared from the source site, so I updated it to a newer version.